src/ui/session/content/chat_action_bar.rs

```diff
@@ -761,7 +766,7 @@ impl ChatActionBar {
             match chat.chat_type() {
                 model::ChatType::Private(user) => {
                     let is_deleted = matches!(user.user_type().0, tdlib::enums::UserType::Deleted);
-                    let is_blocked = chat.is_blocked();
+                    let is_blocked = matches!(chat.block_list(), Some(model::BoxedBlockList(_)));
```

Comment:
I think this can be made a function `is_blocked()` in `Chat`.

Reply:
Right, while also keeping the `block_list` for future use.